.devcontainer/devcontainer.json

```diff
@@ -18,7 +18,7 @@
     "ghcr.io/devcontainers/features/sshd:1": {},
     "ghcr.io/devcontainers/features/github-cli:1": {},
     "ghcr.io/devcontainers/features/java:1.5.0": {
-      "version": "11",
+      "version": "17",
```

Comment:
@dkatzz Is there anything else involved in making the devcontainers update to Java 17? Aside from the config, I assume people will need to destroy and create new instances.

Reply:
That's good to know. Losing bash history isn't the end of the world.